recipes/ectyper/meta.yaml

```diff
@@ -15,8 +15,8 @@ source:
     sha256: 7328c3c5e9c92c3626e762e87232358af2433f675b8c8ca4a75868f388e37750
 
 build:
-    skip: True # [py27, py34]
-    number: 7
+    skip: True # [py27]
```

Comment:
You may remove this line.

Reply:
Thank you for looking at this. The package requires at least Python 3.5, but I can't seem to specify this correctly in the build options.